src/main/kotlin/com/hypto/iam/server/authProviders/MicrosoftAuthProvider.kt

```diff
@@ -16,7 +16,7 @@ import org.koin.core.component.KoinComponent
 import org.koin.core.component.inject
 import org.koin.core.qualifier.named
 
-object MicrosoftAuthProvider : BaseAuthProvider, KoinComponent {
+object MicrosoftAuthProvider : BaseAuthProvider(), KoinComponent {
```

Comment:
suggestion: 
You can accept `isVerifiedProvider` and `providerName` as constructor arguments (isVerifiedProvider being optional defaulting to true) and simply pass the values upon initialization

```suggestion
object MicrosoftAuthProvider : BaseAuthProvider("microsoft", false), KoinComponent {
```

Reply:
Addressed